### PR TITLE
Add `osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park` wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
Wiki article for the recently concluded `osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park` LAN tournament.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+4. All players are required to attend the technical meeting held onlint by the Tournament Management before the tournament commences.
+5. Test players, referees, and mappool selectors may not participate as players in this tournament.
+
+### Match instructions
+
+1. Each player must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets to pick first.
+   - The loser of the `!roll` gets the opposite by default.
+2. EMap banning **does not apply** in the entirety of the tournament.
+3. **There will be no warm-up beatmaps to be played in the multiplayer lobby.** Players who are looking up to warm themselves up are expected to do so by their own during the grace period in between the matches.
```
Comment (Blooshing) [inline wiki/Tournaments/MUKASHI/1/en.md:208]:
``` Suggestion
3. **There will be no warm-up beatmaps to be played in the multiplayer lobby.** Players who are looking to warm up are expected to do so by their own during the grace period in between the matches.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 300,000 IDR, commemorative certificate |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 200,000 IDR, commemorative certificate |
+
+## Organisation
+
+The osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park was run by various POST representatives and Indonesian osu! community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ::{ flag=ID }:: Raben (POST), ::{ flag=ID }:: Ijul (POST), ::{ flag=ID }:: Erlangga (POST) |
```
Comment (TicClick) [inline wiki/Tournaments/MUKASHI/1/en.md:46]: is there any particular reason their profiles aren't linked, although at least two of them [appear](https://osu.ppy.sh/users/31664650) to [exist](https://osu.ppy.sh/users/30984728)?

**Review by TicClick — COMMENTED** [review]
looks good to me, other than that

File: wiki/Tournaments/MUKASHI/1/en.md (hunk 2 above)
Comment (Nivalyx) [reply wiki/Tournaments/MUKASHI/1/en.md:46]: @TicClick those are not "Ijul" and "Erlangga" that are mentioned in the article lol

Comment (Nivalyx) [reply wiki/Tournaments/MUKASHI/1/en.md:46]: (i genuinely dunno their osu! account or if any of them even have an osu! account tbh)

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+| Commentator | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805), ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677), ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) |
+| Referee | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |
+| Documentation | ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677), ::{ flag=ID }:: [liykun](https://osu.ppy.sh/users/9500057) |
+| Wiki editor | ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) |
+| Additional helper | ::{ flag=ID }:: [Ascaveth](https://osu.ppy.sh/users/3245206), ::{ flag=ID }:: [Suikami](https://osu.ppy.sh/users/1929336) |
+
+## Links
+
+- **[Technical meeting document](https://docs.google.com/presentation/d/14yBMl4KiTAYxBxLrECJgixUlQfMkcKrQsYNi5wx-oOs/edit#slide=id.p)**
+- [POST website](https://santai.web.id/)
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:59]: website says "POS" and not "POST"?

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Team |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=ID }:: **[LuxKooky](https://osu.ppy.sh/users/4691146)** |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=ID }:: **[\[ -Asriel- \]](https://osu.ppy.sh/users/11829623)** |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=ID }:: **[KinoJourney](https://osu.ppy.sh/users/15798093)**  |
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:85]:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=ID }:: [LuxKooky](https://osu.ppy.sh/users/4691146) |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=ID }:: [\[ -Asriel- \]](https://osu.ppy.sh/users/11829623) |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=ID }:: [KinoJourney](https://osu.ppy.sh/users/15798093)  |
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 5]
```diff
@@ -0,0 +1,214 @@
...
+
+| Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | **[LuxKooky](https://osu.ppy.sh/users/4691146)** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: [\[ -Asriel- \]](https://osu.ppy.sh/users/11829623) | [#1](https://osu.ppy.sh/community/matches/105421255) |
+
+### 3rd Place Match
+
+| Match code  | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| 3rd Place Match | [\[ID\]Xenotia-](https://osu.ppy.sh/users/13865224) ::{ flag=ID }:: | 2 | **4** | ::{ flag=ID }:: **[KinoJourney](https://osu.ppy.sh/users/15798093)** | [#1](https://osu.ppy.sh/community/matches/105421643) |
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:150]:
```suggestion
| 3rd-place match | [\[ID\]Xenotia-](https://osu.ppy.sh/users/13865224) ::{ flag=ID }:: | 2 | **4** | ::{ flag=ID }:: **[KinoJourney](https://osu.ppy.sh/users/15798093)** | [#1](https://osu.ppy.sh/community/matches/105421643) |
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+## Ruleset
+
+### General rules
+
+1. Beatmap scoring is based on **[ScoreV2](/wiki/Gameplay/Score#scorev2).**
+2. All matches will be played over a single day in at [Senayan Park, Central Jakarta](https://senayanpark.com/en/) (as part of MUKASHI FESTIVAL ID) following the Single Elimination system.
+   - The bracket for the tournament will be randomly generated using [random.org](https://random.org) during the technical meeting.
+3. All matches will be played using the setup provided by the Tournament Management on the spot.
+   - Players are allowed to bring in certain equipments (e.g. keyboard and laptop) and their own device settings should they wish to.
+4. All maps and mappools for the tournament will be announced by the Tournament Management in advance before the event takes place.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:183]:
```suggestion
4. All maps and mappools for the tournament will be announced by the tournament management in advance before the event takes place.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 7]
```diff
@@ -0,0 +1,214 @@
...
+5. Match schedules will be predetermined by the Tournament Management.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:184]:
```suggestion
5. Match schedules will be predetermined by the tournament management.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 8]
```diff
@@ -0,0 +1,214 @@
...
+6. A referee will create a multiplayer room 10 minutes in advance and will start to send out invites.
+7. If a player disconnects, the referee will note the score the player had achieved at the end of the beatmap directly from their PC and adjust the outcome of the pick accordingly.
+8. Lag is not a valid reason to nullify a beatmap.
+9. If any problems during the match occur, the Tournament Management will make a decision based on the referee's report.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:188]:
```suggestion
9. If any problems during the match occur, the tournament management will make a decision based on the referee's report.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 9]
```diff
@@ -0,0 +1,214 @@
...
+10. It is expected that all players be polite and respectful to each other. Penalties will be given upon violation.
+    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the Tournament Management.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:190]:
```suggestion
    - If a player is found to be engaging in an act that is deemed to be distasteful or provocative, the corresponding player may be disqualified right away from the tournament and/or blacklisted from future iterations of the tournament by the tournament management.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 10]
```diff
@@ -0,0 +1,214 @@
...
+    - Usage of any tools or programs that are against the [osu! community rules](/wiki/Rules#community-rules) is strictly prohibited and will be straight up reported to the osu! team at will.
+
+### Tournament registration
+
+1. All interested players are required to register into the tournament individually via [this form](https://docs.google.com/forms/d/e/1FAIpQLSfM4bHa5fqs5f9O-bb4SHqqPDTUjNjYvHzDNrZH3siRj_9hyA/closedform).
+2. To ensure valid and serious registrations, every registered player will be checked by the Tournament Management.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:196]:
```suggestion
2. To ensure valid and serious registrations, every registered player will be checked by the tournament management.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 11]
```diff
@@ -0,0 +1,214 @@
...
+3. The list of players who are deemed to be eligible to compete in the tournament will be published by the Tournament Management after the Registration phase has ended.
+   - Due to time constraints, the Tournament Management would only allow for around 8 players to participate in this tournament.
+4. All players are required to attend the technical meeting held online by the Tournament Management before the tournament commences.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:199]:
```suggestion
3. The list of players who are deemed to be eligible to compete in the tournament will be published by the tournament management after the registration phase has ended.
   - Due to time constraints, the tournament management will only allow for around 8 players to participate in this tournament.
4. All players are required to attend the technical meeting held online by the tournament management before the tournament commences.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 12]
```diff
@@ -0,0 +1,214 @@
...
+5. Test players, referees, and mappool selectors may not participate as players in this tournament.
+
+### Match instructions
+
+1. Each player must use the `!roll` command once in the multiplayer lobby in order to determine the banning and picking order.
+   - The winner of the `!roll` gets to determine who gets to pick first.
+   - The loser of the `!roll` gets the opposite by default.
+2. Map banning **does not apply** in the entirety of the tournament.
+3. **There will be no warm-up beatmaps to be played in the multiplayer lobby.** Players who are looking to warm up are expected to do so by their own during the grace period in between the matches.
+4. Although playing with mods is not mandatory, **all maps (including the tiebreaker) will be played with the FreeMod option enabled.**
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:209]:
```suggestion
4. Although playing with mods is not mandatory, **all maps (including the tiebreaker) will be played with the Free Mod option enabled.**
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 13]
```diff
@@ -0,0 +1,214 @@
...
+   - Players are allowed to play any map with NoFail, Hidden, FadeIn, Flashlight, and Mirror should they wish to.
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:210]:
```suggestion
   - Players are allowed to play any map with No Fail, Hidden, Fade In, Flashlight, and Mirror should they wish to.
```

Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:210]:
```suggestion
   - Players are allowed to play any map with No Fail, Hidden, Fade In, Flashlight, and Mirror should they wish to.
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 14]
```diff
@@ -0,0 +1,214 @@
...
+5. Players who can compete in the next round are determined by:
+   - In the Extra Preliminary Match, the Quarterfinals, and the Semifinals, each player needs to win 3 points in order to win a match. (Best-of-5)
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:212]:
```suggestion
   - In the extra preliminary match, the Quarterfinals, and the Semifinals, each player needs to win 3 points in order to win a match. (Best of 5)
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 15]
```diff
@@ -0,0 +1,214 @@
...
+   - In the 3rd Place Match and the Grand Final, each player needs to win 4 points in order to win a match. (Best-of-7)
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:213]:
```suggestion
   - In the 3rd-place match and the Grand Final, each player needs to win 4 points in order to win a match. (Best of 7)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 16]
```diff
@@ -0,0 +1,214 @@
...
+tags:
+  - MUKASHI
+  - MUKASHI SPARK
+---
+
+# osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park
+
+![osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park logo](img/logo.jpg)
+
+The **osu!mania 4K Community Tournament @ MUKASHI FESTIVAL ID Senayan Park** was a single-elimination 1v1 osu!mania LAN tournament hosted by [Perkumpulan Orang Santai Total (POST)](https://santai.web.id/) under the guidance and supervision of ::{ flag=ID }:: [Niva](https://osu.ppy.sh/users/197805) and ::{ flag=ID }:: [Vvn](https://osu.ppy.sh/users/3204677) from the Indonesian osu! community. It was the first-ever osu!mania tournament of this scale to be held by POST, who had previously facilitated tournaments for other games such as Tekken 7.
```
[the remark this answers is not in the document]
Comment (Nivalyx) [reply wiki/Tournaments/MUKASHI/1/en.md:11]: yeah POST is a community/organization of some sort

File: wiki/Tournaments/MUKASHI/1/en.md (hunk 3 above)
Comment (Nivalyx) [reply wiki/Tournaments/MUKASHI/1/en.md:59]: yea to my knowledge there are instances where they went with `POS` `POST` and even `POS Total`

i think i'll keep everything here in the article as `POST` for consistency tho

File: wiki/Tournaments/MUKASHI/1/en.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (Walavouchey) [reply wiki/Tournaments/MUKASHI/1/en.md]: ah i'm blind

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 17]
```diff
@@ -0,0 +1,214 @@
...
+
+| Placing | Team |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=ID }:: [LuxKooky](https://osu.ppy.sh/users/4691146) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=ID }:: [\[ -Asriel- \]](https://osu.ppy.sh/users/11829623) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=ID }:: [KinoJourney](https://osu.ppy.sh/users/15798093)  |
+
+## Mappools
+
+### Grand Final and 3rd Place Match
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:89]:
```suggestion
### Grand Final and 3rd-place match
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 18]
```diff
@@ -0,0 +1,214 @@
...
+  3. [positive MAD-Crew - Mynarco (Mipha-) \[Addiction\]](https://osu.ppy.sh/beatmapsets/1664054#osu/3397069)
+- LN
+  1. [SYUNN - Sulk (stupud man) \[Stage 3: Bittersweet\]](https://osu.ppy.sh/beatmapsets/1294885#mania/2686796)
+  2. [ZYTOKINE - DESIRE DREAM feat. itori - FELT Remix (\_underjoy) \[Insane Side B\]](https://osu.ppy.sh/beatmapsets/836634#mania/1761189)
+- Hybrid
+  1. [Wednesday Campanella - Shakushain (Penguinosity) \[Shakushaku 1.1x\]](https://osu.ppy.sh/beatmapsets/1503374#mania/3079991)
+- Tiebreaker
+  1. **[technoplanet - Megastructure \[Extended Mix\] (Nimis) \[Croco's Insane\]](https://osu.ppy.sh/beatmapsets/1568277#mania/3205391)**
+
+### Extra Preliminary Match and Quarterfinals
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:122]:
```suggestion
### Extra preliminary match and Quarterfinals
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 19]
```diff
@@ -0,0 +1,214 @@
...
+
+## Match results
+
+### Grand Final
+
+| Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | **[LuxKooky](https://osu.ppy.sh/users/4691146)** ::{ flag=ID }:: | **4** | 1 | ::{ flag=ID }:: [\[ -Asriel- \]](https://osu.ppy.sh/users/11829623) | [#1](https://osu.ppy.sh/community/matches/105421255) |
+
+### 3rd Place Match
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:146]:
```suggestion
### 3rd-place match
```

File: wiki/Tournaments/MUKASHI/1/en.md [hunk 20]
```diff
@@ -0,0 +1,214 @@
...
+### Quarterfinals
+
+| Match code | Player 1 |  |  | Player 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Quarterfinal 1 | **[LuxKooky](https://osu.ppy.sh/users/4691146)** ::{ flag=ID }:: | **3** | 0 | ::{ flag=ID }:: [youtoo12](https://osu.ppy.sh/users/5110129) | [#1](https://osu.ppy.sh/community/matches/105418603) |
+| Quarterfinal 2 | [anteiku7](https://osu.ppy.sh/users/7013247) ::{ flag=ID }:: | 0 | **3** | ::{ flag=ID }:: **[KinoJourney](https://osu.ppy.sh/users/15798093)** | [#1](https://osu.ppy.sh/community/matches/105418912) |
+| Quarterfinal 3 | **[\[ID\]Xenotia-](https://osu.ppy.sh/users/13865224)** ::{ flag=ID }:: | **3** | 0 | ::{ flag=ID }:: [Fansly](https://osu.ppy.sh/users/10410817) | [#1](https://osu.ppy.sh/community/matches/105419216) |
+| Quarterfinal 4 | [KarinaKatsuna](https://osu.ppy.sh/users/31669247) ::{ flag=ID }:: | 0 | **3** | ::{ flag=ID }:: **[\[ -Asriel- \]](https://osu.ppy.sh/users/11829623)** | [#1](https://osu.ppy.sh/community/matches/105419558) |
+
+### Extra Preliminary Match
```
Comment (Walavouchey) [inline wiki/Tournaments/MUKASHI/1/en.md:168]:
```suggestion
### Extra preliminary match
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**